targets/lenovo-x1-carbon.nix

```diff
@@ -118,6 +118,11 @@
         # Early KMS needed for GNOME to work inside GuiVM
         boot.initrd.kernelModules = ["i915"];
 
+        # Enable pulseaudio for user ghaf
+        sound.enable = true;
+        hardware.pulseaudio.enable = true;
+        users.extraUsers.ghaf.extraGroups = ["audio"];
```

Comment:
Hmm, I don't think there belong here? Have you rebased correctly with the `main`? Ping @josa41

Reply:
Yeah the current plan is that guivm should not use audio device directly

Reply:
It is removed.